datafusion/datasource-parquet/src/mod.rs

```diff
@@ -24,7 +24,7 @@ pub mod file_format;
 mod metrics;
 mod opener;
 mod page_filter;
-mod reader;
+pub mod reader;
```

Comment:
why does this need to be made pub? I reverted the change and things still seem to compile just fine

Reply:
I see -- my concern is that this change might be easy to accidentally undo / break in the future. Maybe to make it more deliberate, you could leave `mod reader` and then `pub use` both `CachedParquetMetaData` and `CachedParquetMetaData` 🤔